src/components/sky-nav/sky-nav.twig

```diff
@@ -0,0 +1,40 @@
+{# https://webaim.org/techniques/skipnav/ #}
+<a href="#main">
+  Skip to main content
+</a>
+{#
+  This may require `role="banner"` if nested in something other than `body`.
+  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_Role
+#}
+<header>
+  <a href="/"
+    {% if is_home %}
+      aria-current="page"
+    {% else %}
+      title="Return to home"
+    {% endif %}>
+    {# https://heydonworks.com/article/aria-label-is-a-xenophobe/ #}
+    <span class="u-hidden-visually">Home: Cloud Four</span>
+    {# http://simplyaccessible.com/article/7-solutions-svgs/#acc-heading-4 #}
+    {% include '../../assets/icons/cloud-four.svg.twig' with {
+      width: '80',
+      height: '80',
+      focusable: 'false'
+    } only %}
+  </a>
+  <button type="button">
+    <span class="u-hidden-visually">Toggle Main</span>
+    Menu
+  </button>
+  <nav aria-label="Main Menu">
```

Comment:
I know I've implemented a similar markup pattern in a past project, but now that I'm reviewing again I wonder whether it would be better to use `aria-labelledby`? I'm curious what others think.

**Pros:**

- `aria-labelledby` is more reliably translated by translation tools

**Cons:**

- `aria-labelledby` would require an additional HTML element which screen readers would need to read. It might be overly verbose.

---

I'm not sure what the "right" answer is here, but figured I'd mention my thoughts in case others had more insight or felt strongly.